hack/lib/tracing.bash

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+function install_tracing {
+  deploy_zipkin
+  enable_eventing_tracing
+}
+
+function deploy_zipkin {
+  logger.info "Installing Zipkin in namespace ${ZIPKIN_NAMESPACE}"
+  cat <<EOF | oc apply -f - || return $?
+apiVersion: v1
+    kind: Service
+    metadata:
+      name: zipkin
+      namespace: ${ZIPKIN_NAMESPACE}
+    spec:
+      type: NodePort
+      ports:
+      - name: http
+        port: 9411
+      selector:
+        app: zipkin
+    ---
+    apiVersion: apps/v1
```

Comment:
Fix the indentation here? I believe it can all start at the left-most character.